src/datasources/exchange-api/exchange-api.service.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { ExchangeApi } from './exchange-api.service';
+import { ExchangeResult } from '../../domain/exchange/entities/exchange-result.entity';
+import { FiatCodesExchangeResult } from '../../domain/exchange/entities/fiat-codes-result.entity';
+import { FakeConfigurationService } from '../../common/config/__tests__/fake.configuration.service';
+import { mockNetworkService } from '../../datasources/network/__tests__/test.network.module';
+
+const FIATCODES: FiatCodesExchangeResult = {
+  success: true,
+  symbols: {},
+};
+
+const EXCHANGERESULT: ExchangeResult = {
+  success: true,
+  rates: {},
+  base: '',
+};
```

Comment:
I would move this to inside each test. Tests referencing a more global context are more confusing to read in my opinion as any potential state change might affect the outcome of the tests.

You can use `exchangeResultFactory` in each test to get a `ExchangeResult`

Reply:
I haven't a strong opinion about this, I agree with you :)